src/client/systems/sysstarsystemrenderer.cpp

```diff
@@ -106,6 +114,8 @@ void conquerspace::client::systems::SysStarSystemRenderer::Render() {
         m_viewing_entity = current_planet;
         // Do terrain
         SeeEntity();
+    } else if (current_planet != entt::null) {
+        //view_center = CalculateObjectPos(m_viewing_entity);
```

Comment:
Why comment this? Try not to do things with updating the renderer on the scene. I want to limit the work done in updating the things that go on in the systems. Calculating the object position is trivial.